packages/suite/src/views/wallet/staking/components/StakingDashboard/components/StakingCard.tsx

```diff
@@ -233,22 +245,34 @@ export const StakingCard = ({
                         />
                     )}
 
-                    <Item
-                        label={
-                            <Translation
-                                id={
-                                    isCardanoNetworkType
-                                        ? 'TR_STAKE_ACTIVE_STAKE'
-                                        : 'TR_STAKE_STAKE'
-                                }
-                            />
-                        }
-                        iconName="lock"
-                        symbol={selectedAccount?.symbol}
-                        cryptoAmount={depositedBalance || '0'}
-                        fiatAmount={depositedBalance || '0'}
-                        data-testid="@account/staking/staked"
-                    />
+                    {isCardanoNetworkType ? (
+                        <Item
+                            label={<Translation id="TR_STAKE_STAKED_AUTOMATICALLY" />}
+                            iconName="check"
+                            symbol={selectedAccount?.symbol}
+                            title={<Translation id="TR_STAKE_FULL_BALANCE" />}
+                            description={
+                                <Translation
+                                    id="TR_STAKE_FUNDS_FULLY_ACCESSIBLE"
+                                    values={{
+                                        networkDisplaySymbol: getNetworkDisplaySymbol(
+                                            selectedAccount.symbol,
+                                        ),
+                                    }}
+                                />
+                            }
+                            data-testid="@account/staking/full-balance"
+                        />
+                    ) : (
+                        <Item
+                            label={<Translation id="TR_STAKE_STAKE" />}
+                            iconName="lock"
+                            symbol={selectedAccount?.symbol}
+                            cryptoAmount={depositedBalance || '0'}
+                            fiatAmount={depositedBalance || '0'}
```

Comment:
pass formatted crypto amount and base currency value here using title and description